packages/components/src/components/Log/Log.jsx

```diff
@@ -195,6 +189,7 @@ export class LogContainer extends Component {
   };
 
   getScrollButtons = () => {
+    const carbonPrefix = this.context;
```

Comment:
Any difference/benefit between using `PrefxContext` and `useContext`?

Reply:
`useContext` can't be used in a class-based component, so we rely on the static `contextType` instead.

We use the `usePrefix` hook in any of the functional component that need this value.